cmd_clean/main.go

```diff
@@ -62,8 +64,20 @@ func main() {
 	cf.Cluster = cluster
 
 	logf.SetLogger(logger.ZapLogger(cf.DefaultConfig.Debug, config.LogLevel))
+	// just a demo to show how to use customer http client
```

Comment:
Is this a demo, or can it actually be run this way?

If just a demo, maybe this should be moved to a `demo` package or something so user's would not intentionally run this as a script/binary?

Reply:
It actually be run this way, removed the `demo` comment